### ISSUE TITLE
Copying example verbatim from example shows error

### ISSUE THREAD

**Issue by peterolson** [threlte/threlte#1086]
I copied the code verbatim from this page:

https://threlte.xyz/docs/reference/extras/meshline-geometry

but I just see an empty grid rotating with no line

![image](https://github.com/user-attachments/assets/45e23f3d-a42c-4e7a-ae98-906dba5ed0a2)

and I see the following error in my console:

```
root.svelte:21 THREE.WebGLProgram: Shader Error 0 - VALIDATE_STATUS false

Material Name: 
Material Type: ShaderMaterial

Program Info Log: Fragment shader is not compiled.


FRAGMENT

ERROR: 0:211: 'LinearTosRGB' : no matching overloaded function found
ERROR: 0:211: '=' : dimension mismatch
ERROR: 0:211: 'assign' : cannot convert from 'const mediump float' to 'out highp 4-component vector of float'


  206: 
  207: 	if( useDash == 1. ){
  208: 			c.a *= ceil(mod(vCounters + dashOffset, dashArray) - (dashArray * dashRatio));
  209: 	}
  210: 
> 211: 	gl_FragColor = LinearTosRGB(c);
  212: }
  213: 

255
WebGL: INVALID_OPERATION: useProgram: program not valid
root.svelte:21 WebGL: INVALID_OPERATION: drawElements: no valid shader program in use
chunk-GMFGCRRE.js?v=676ebfb3:14301 WebGL: too many errors, no more errors will be reported to the console for this context.
﻿
```

## Steps to reproduce:

Clone
https://github.com/peterolson/asl-pose-svelte4

`npm install` and `npm run dev`

---
Alternatively, press the "StackBlitz" button on https://threlte.xyz/docs/reference/extras/meshline-geometry

**Comment by jkufa** [comment]
Hey @peterolson, I ran into a similar issue today trying out the same code example. I was on the latest version of `three: "^0.168.0"`, but changing the version to `"three": "^0.160.1",` (the same version in @threlte/extras's [package.json](https://github.com/threlte/threlte/blob/main/packages/extras/package.json)) solve it for me.

**Comment by joshwashywash** [comment]
this should be closed when #1097 gets merged. see ticket #1095 for details.

**Comment by grischaerbe** [comment]
This should be fixed, thanks all!
